### PR TITLE
Fix deprecated code 

### PR DESCRIPTION
**Pull request by tyxeron**
Fixed issue the following issue:

 'asn1crypto.keys.PublicKeyInfo().fingerprint has been removed, please use oscrypto.asymmetric.PublicKey().fingerprint instead'

### CODE REVIEW

File: androguard/cli/main.py [hunk 1]
```diff
@@ -423,6 +423,7 @@ def androsign_main(args_apk, args_hash, args_all, show):
     import traceback
     from colorama import Fore, Style
     from asn1crypto import x509, keys
+    from oscrypto import asymmetric
```
Comment (jspricke) [inline androguard/cli/main.py:426]: please add oscrypto to the requirements.txt

**Review by jspricke — APPROVED** [review]
I have tested this successfully.

File: androguard/cli/main.py (hunk 1 above)
Comment (tyxeron) [reply androguard/cli/main.py:426]: will do

**Comment by totoag** [comment]
I removed these piece of code

**Pull request closed without merging**